src/docs/sdk/sessions.mdx

```diff
@@ -307,17 +325,18 @@ The most basic API exposed is on the hub level and lets you start and stop sessi
 
 `Hub.start_session()`
 
-> Stores a session on the current scope and starts tracking it. This normally attaches a brand new `session` to the scope.
+> Stores a session on the current scope and starts tracking it. This normally attaches a brand new `session` to the scope,
```

Comment:
Are the backticks on the second instance (but not first or third instances) of the word "session" there to imply that the scope ends up with a property called `.session`? Also: "normally" - are there cases where this doesn't happen?